dashboard/src/main/webapp/src/components/CoordinatorServerPage.vue

```diff
@@ -0,0 +1,149 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<template>
+  <div class="demo-collapse">
+    <el-collapse v-model="pageData.activeNames" accordion:false>
+      <el-collapse-item title="Coordinator Server" name="1">
```

Comment:
This title is not necessary, all the page is under the coordinator server page.

Reply:
> This title is not necessary, all the page is under the coordinator server page.

The page has a collapsible effect. We need to add this title here.